.gitlab/JOBOWNERS

```diff
@@ -15,8 +15,7 @@ cluster_agent_cloudfoundry-build*    @Datadog/integrations-tools-and-libs
 cluster_agent-build*                 @DataDog/container-integrations
 
 # Kitchen testing
-# Don't notify on kitchen Windows tests since they are too flaky
-kitchen_windows_*                    @DataDog/do-not-notify
+kitchen_windows_*                    @DataDog/agent-platform
```

Comment:
We already have a catch-all pattern in the first line
https://github.com/DataDog/datadog-agent/blob/30a411718c421d286a862b299e09b0e1a4cf2aba/.gitlab/JOBOWNERS#L1

so we could just remove it if we want it. I am fine either way

Reply:
Oh, I didn't realize that. Removing is better, I'll do that :)